interp/impargs.mli

```diff
@@ -138,4 +138,5 @@ val select_impargs_size : int -> implicits_list list -> implicit_status list
 val select_stronger_impargs : implicits_list list -> implicit_status list
 
 val explicitation_eq : Constrexpr.explicitation -> Constrexpr.explicitation -> bool
+  [@@deprecated "Use Constrexpr_ops.explicitation_eq instead (since 8.10)"]
```

Comment:
I do believe that the preferred form of the attribute is `ocaml.deprecated`

Reply:
Done.